### PR TITLE
Edit symbols

### PR DESCRIPTION
**Pull request by guillermooo**
Made a few edits. I'll revise other metadat topics too.

### CODE REVIEW

**Comment by guillermooo** [comment]
@FichteFoll PTAL

File: source/reference/symbols.rst [hunk 1]
```diff
@@ -41,19 +42,30 @@ Several symbol definition files can coexist
...
+is strictly text-based
+and uses scope selectors
 to capture symbols in source code files.
+No lexical or syntactical analysis is performed.
+
+Sublime Text features two types of symbol list:
+a local symbol list (active file)
+and a global symbol list (project-wide).
+Using symbol definition files,
+you can target the two of them.
```
Comment (FichteFoll) [inline source/reference/symbols.rst]: +individually

File: source/reference/symbols.rst [hunk 2]
```diff
@@ -185,59 +199,59 @@ which is inherited from the Property List format.
...
    Optional.
-
-   ``0`` or ``1`` (unlike other settings).
+   Links symbols to the local symbol list.
+   Valid values are ``0`` or ``1``.
    If ``0``,
-   the corresponding symbols will be hidden instead of indexed.
-
+   the corresponding symbols
+   will be hidden instead of indexed.
```
Comment (FichteFoll) [inline source/reference/symbols.rst]: I actually prefer the term `indexed` for the indexed symbol list, the project one. Maybe "collected"?

File: source/reference/symbols.rst [hunk 3]
```diff
@@ -185,59 +199,59 @@ which is inherited from the Property List format.
...
-         ...
-         <key>symbolTransformation</key>
-         <string>
-         s/class\s+([A-Za-z_][A-Za-z0-9_]*.+?\)?)(\:|$)/$1/g;
-         s/def\s+([A-Za-z_][A-Za-z0-9_]*\()(?:(.{0,40}?\))|((.{40}).+?\)))(\:)/$1(?2:$2)(?3:$4…\))/g;
-         </string>
-      </dict>
+      <key>symbolTransformation</key>
+      <string>
+      s/class\s+([A-Za-z_][A-Za-z0-9_]*.+?\)?)(\:|$)/$1/g;
```
Comment (FichteFoll) [inline source/reference/symbols.rst]: How about indenting this btw?

And I also just realized that whitespace between two substitution instructions is ignored which we don't mention.

File: source/reference/symbols.rst [hunk 4]
```diff
@@ -185,59 +199,59 @@ which is inherited from the Property List format.
...
+      s/def\s+([A-Za-z_][A-Za-z0-9_]*\()(?:(.{0,40}?\))|((.{40}).+?\)))(\:)/$1(?2:$2)(?3:$4…\))/g;
+      </string>
 
 ``symbolIndexTransformation``
    Optional.
-   Similar to ``symbolTransformation``
-   but modifies the project symbol list.
+   Targets the global symbol list.
+   Semicolon-separated list of text transformations
```
Comment (FichteFoll) [inline source/reference/symbols.rst]: I prefer the usage of "substitutions" here.

That means the instructions are called a "transformation" collectively but consist of "substitutions".

**Comment by FichteFoll** [comment]
Otherwise LGTM.

File: source/reference/symbols.rst (hunk 1 above)
Comment (guillermooo) [inline source/reference/symbols.rst]: ok

File: source/reference/symbols.rst (hunk 2 above)
Comment (guillermooo) [inline source/reference/symbols.rst]: reworded so I don't need to mention that.

File: source/reference/symbols.rst (hunk 3 above)
Comment (guillermooo) [inline source/reference/symbols.rst]: Changed. Hope I got it right.

File: source/reference/symbols.rst (hunk 4 above)
Comment (guillermooo) [inline source/reference/symbols.rst]: I like that.

Comment (guillermooo) [inline source/reference/symbols.rst]: Changed that in other places that seemed to make sense. Take a look at them in the published version when you can to see if they need to be reverted.

**Pull request merged**